custom-metrics-stackdriver-adapter/examples/cross-project-metrics/README.md

```diff
@@ -0,0 +1,169 @@
+# Custom Metrics with Stackdriver
+
+This is a minimal example showing how to use the stackdriver adapter to get metrics from
+a different project than the one your cluster is running in. By default, the adapter
+uses the credentials of the node machine. There are three approaches for granting permissions
+to your other project:
+
+
+1. Grant the default service account for your project access to `roles/monitoring.viewer` in
+   your other project.
+
+1. Create your GKE cluster with a service account, and grant the service account access to 
+   `roles/monitoring.viewer` in your other project
+
+1. (Recommended) Create a service a service account for the stackdriver-adapter, and mount
+   the credentials into the pod. This is the method this example implements.
+
+
+This example assumes you already have custom metrics set up in your cluster
+
+## Create a service account with credentials
+
+For this example, you will need to create a service account. See [Creating a service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
+for how to do that.
+
+
+Next, you will need to create a key for that services account. See [Creating service account keys](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console) for how to do that.
+
+
+Copy the key to this directory. The filename should be `key.json`
+
+
+Add `roles/monitoring.viewer` to your service account in all projects that you want it to grab metrics from.
+
+## Creating the kubernetes secret
+
+Run the following command:
+
+```
+kubectl create secret generic custom-metrics-stackdriver-adapter --from-file=./key.json -n custom-metrics
+```
+
+
+## Apply the modified adapter
+
+Run the following command:
+
+```
+kubectl apply -f modified-adapter.yml
```

Comment:
Can you add a link using raw github content instead of assuming that user has cloned this repo? Something like: https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/examples/cross-project-metrics/adapter.yaml